docs/content/_index.md

```diff
@@ -651,22 +651,27 @@ On Linux (64-bit):
 curl -L -o opa https://openpolicyagent.org/downloads/{{< current_version >}}/opa_linux_amd64_static
 ```
 
-> Windows users can obtain the OPA executable from [here](https://openpolicyagent.org/downloads/{{< current_version >}}/opa_windows_amd64.exe).
-> The steps below
-> are the same for Windows users except the executable name will be different.
+{{< info >}}
+Windows users can obtain the OPA executable from [here](https://openpolicyagent.org/downloads/{{< current_version >}}/opa_windows_amd64.exe).
```

Comment:
Maybe we should mention using WSL as an option too.

Reply:
Probably... but I'm not competent with that

Reply:
Yeah it can wait until we do a [concentrated effort](https://github.com/open-policy-agent/opa/issues/4193) on Windows options.